### DISCUSSION TITLE
Docker Hub - publisher - Trusted Content

### DISCUSSION THREAD

**Discussion by wrutkowski-xebia** [Azure/data-api-builder#2158] [Q&A] [answered]
Hello,

Why image on Docker Hub is not published by Verified Publisher or Docker Official Image?

`mcr.microsoft.com/azure-databases/data-api-builder:tag`
`https://hub.docker.com/_/microsoft-azure-databases-data-api-builder`

![image](https://github.com/Azure/data-api-builder/assets/112972470/319722dd-34b8-4424-a98f-ac95980aa1ce)

**Comment by seantleonard** [comment]
Thank you for raising this, we have this on our radar now, and will be looking into this.

**Reply by seantleonard** [reply, marked as answer]
created issue #2173 for tracking.,
